src/Sylius/Bundle/CoreBundle/Fixture/Factory/ShopUserExampleFactory.php

```diff
@@ -99,6 +103,24 @@ protected function configureOptions(OptionsResolver $resolver): void
             ->setDefault('customer_group', LazyOption::randomOneOrNull($this->customerGroupRepository, 100))
             ->setAllowedTypes('customer_group', ['null', 'string', CustomerGroupInterface::class])
             ->setNormalizer('customer_group', LazyOption::findOneBy($this->customerGroupRepository, 'code'))
+            ->setDefault('gender', CustotmerComponent::UNKNOWN_GENDER)
+            ->setAllowedValues(
+                'gender',
+                [CustotmerComponent::UNKNOWN_GENDER, CustotmerComponent::MALE_GENDER, CustotmerComponent::FEMALE_GENDER]
+            )
+            ->setDefault('phone_number', function (Options $options): string {
+                return $this->faker->phoneNumber;
+            })
+            ->setDefault('birthday', function (Options $options): \DateTime {
+                return $this->faker->dateTimeThisCentury();
+            })
+            ->setAllowedTypes('birthday', ['null', 'string', \DateTime::class])
```

Comment:
I'd go for `\DateTimeInterface::class` so we can also accept `\DateTimeImmutable`.

Reply:
I will fix that.